.all-contributorsrc

```diff
@@ -670,7 +670,9 @@
       "name": "Christine Belzie",
       "avatar_url": "https://avatars.githubusercontent.com/u/105683440?v=4",
       "profile": "https://linkfree.eddiehub.io/CBID2",
-      "contributions": [
```

Comment:
It looks like you're removing Christine's contributions. Can you update this to reflect what she had before?